packages/marko-web-omeda-identity-x/index.js

```diff
@@ -30,6 +30,9 @@ module.exports = (app, params = {}) => {
     omedaPromoCodeCookieName,
     omedaPromoCodeDefault,
     omedaRapidIdentifyProp,
+    onLoginLinkSentFormatter = (async ({ payload }) => ({ ...payload })),
+    onAuthenticationSuccessFormatter = (async ({ payload }) => ({ ...payload })),
+    onUserProfileUpdateFormatter = (async ({ payload }) => ({ ...payload })),
```

Comment:
Define your default formatter above (also no need to spread)
```js
const defaultFormatter = async ({ payload }) => payload;
```
And change this to use the function
```suggestion
    onLoginLinkSentFormatter = defaultFormatter,
    onAuthenticationSuccessFormatter = defaultFormatter,
    onUserProfileUpdateFormatter = defaultFormatter,
```